flow/src/main/java/com/sequenceiq/flow/repository/FlowLogRepository.java

```diff
@@ -76,7 +76,7 @@ public interface FlowLogRepository extends CrudRepository<FlowLog, Long> {
     Set<String> findAllFlowIdsByChainIds(@Param("chainIds") Set<String> chainIds);
 
     @Query("SELECT fl FROM FlowLog fl WHERE fl.flowId IN (:flowIds) ORDER BY fl.created DESC")
-    List<FlowLog> findAllByFlowIdsCreatedDesc(@Param("flowIds") Set<String> flowIds);
+    List<FlowLog> findAllByFlowIdsCreatedDesc(@Param("flowIds") Set<String> flowIds, Pageable pageable);
```

Comment:
have you tried the paging functionality? afaik it only works when repository extends `PagingAndSortingRepository`

Reply:
I have tested it out @horadla23 - it works

Reply:
yes, discussed, thanks!